README.md

```diff
@@ -22,36 +36,248 @@ component written in native code. Using that component from DNX
 currently [requires a workaround][grpc-workaround]; on Windows please run `fix_dnx_grpc.bat`
 before running any gRPC code.
 
-Documentation
--------------
+## Specifying a Project ID
```

Comment:
Gotcha. I'll update to another text and let me know if that works - cause I still want to explain that they need a project id somewhere and allow them to go and create a project if need be in the console.